torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -230,6 +230,11 @@ class TORCH_API RpcAgent {
   std::atomic<std::chrono::milliseconds> rpcTimeout_;
   std::atomic<bool> profilingEnabled_;
   std::shared_ptr<TypeResolver> typeResolver_;
+  // Boolean that indicates whether RpcAgent is running.
+  // Atomic boolean indicating whether this agent is running. It is set in the
+  // child class start() and unset in child class shutdown() and join(). It
+  // controls whether several background threads should be running.
+  std::atomic<bool> rpcAgentRunning_;
```

Comment:
should we explicitly give it a false initial value?

Reply:
Can be done in the constructor initializer list.